scripts/_docker.sh

```diff
@@ -34,17 +34,23 @@ function _docker() {
 
 function _docker_build_with_custom_tag() {
     local repo=$1
+    local env_name=$2
 
     if [[ -z ${repo} ]]; then
         echo "Repo is required" >&2
         return 1
     fi
 
+    if [[ -z ${env_name} ]]; then
```

Comment:
With this if we say:
```
uhd docker build backend
```
The command will try and extract the env name from the `output.json` file as before.

But now if we say:
```
uhd docker build backend abc12345
```
It will cut and deploy the custom backend image to the env abc12345